util/cookies.ts

```diff
@@ -9,9 +9,8 @@ export async function getCookie(name: string) {
 }
 
 export const secureCookieOptions = {
-  httpOnly: true,
-  path: '/',
```

Comment:
Not needed, as this is the default:

- https://github.com/vercel/next.js/pull/72183